internal/api/mfa.go

```diff
@@ -61,6 +69,74 @@ const (
 	QRCodeGenerationErrorMessage   = "Error generating QR Code"
 )
 
+func (a *API) enrollSMSFactor(w http.ResponseWriter, r *http.Request, user *models.User, session *models.Session, params *EnrollFactorParams) error {
```

Comment:
Reverse the order of `r *http.Request` and `w http.ResponseWriter` to follow convention.

Reply:
Actually, looking at this again to confirm do you mean like 

```
enrollSMSFactor(r *http.Request, w http.ResponseWriter
    ... 
)
```

Think the current convention [writer, request 
 though](https://github.com/supabase/auth/blob/master/internal/api/verify.go#L92)

Reply:
Yeah sorry. Ignore it.